source/_integrations/gardena.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: "Gardena"
+description: "Instructions on how to setup Gardena devices within Home Assistant."
+logo: gardena.jpg
+ha_category:
+  - Hub
+  - Vacuum
+  - Sensor
+  - Switch
+ha_release: XXX
+ha_qa_scale: XXX
+ha_config_flow: false
+---
+
+[Gardena API](https://developer.1689.cloud/apis).
+[py-smart-gardena wrapper for gardena API](https://github.com/grm/py-smart-gardena).
+
+There is currently support for the following device types within Home Assistant:
+
+- [Mower](#mower)
+- [Sensor](#sensor)
+- [Switch](#switch) (Power plugs, smart water control, Smart irrigation control)
+
+Entities will automatically be added based on what is declared in the ANdroid/IOS 
+application.
+
+Mowers are added as a vacuum.
+
+## Configuration
+
+You first need to get a client id for your personal installation of gardena. 
+To do so, create an account here : https://developer.1689.cloud/apis
+
+Then you need to create an application and get the client_id as explained here : 
+https://developer.1689.cloud/docs/getting-started
+
```

Comment:
Configuration block is missing, for more information, see: <https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration>